config_examples/station_list.csv.example

```diff
@@ -0,0 +1,5 @@
+station_name,wigos_station_identifier,traditional_station_identifier
+Balaka,0-454-2-AWSBALAKA,AWSBALAKA
```

Comment:
Balaka and Malomo do not have traditional station identifiers, these should be left blank.